src/amo/components/HeroRecommendation/styles.scss

```diff
@@ -82,6 +82,7 @@ $overlayShapeZIndex: 1;
 .HeroRecommendation-heading .AddonTitle {
```

Comment:
btw, you can cut down on selector repetition with an ampersand. It's not a big deal -- not a change request -- just FYI.

```diff
diff --git a/src/amo/components/HeroRecommendation/styles.scss b/src/amo/components/HeroRecommendation/styles.scss
index d4a88c81c..886557729 100644
--- a/src/amo/components/HeroRecommendation/styles.scss
+++ b/src/amo/components/HeroRecommendation/styles.scss
@@ -78,24 +78,26 @@ $overlayShapeZIndex: 1;
   }
 }
 
-.HeroRecommendation-heading,
-.HeroRecommendation-heading .AddonTitle {
-  color: $white;
-  font-size: $font-size-l;
-  font-weight: normal;
-  line-height: 1.188;
-  margin: 14px 0 0 0;
-  overflow-wrap: anywhere;
-
-  @include respond-to(extraExtraLarge) {
-    font-size: $font-size-xl;
-  }
+.HeroRecommendation-heading {
+  &,
+  & .AddonTitle {
+    color: $white;
+    font-size: $font-size-l;
+    font-weight: normal;
+    line-height: 1.188;
+    margin: 14px 0 0 0;
+    overflow-wrap: anywhere;
+
+    @include respond-to(extraExtraLarge) {
+      font-size: $font-size-xl;
+    }
 
-  .AddonTitle-author {
-    &,
-    a,
-    a:link {
-      color: $white;
+    .AddonTitle-author {
+      &,
+      a,
+      a:link {
+        color: $white;
+      }
     }
   }
 }
```